test/quantization/test_quantize_script.py

```diff
@@ -1739,14 +1739,17 @@ def forward(self, x):
                    .run(m.graph)
 
 
-    def test_insert_quant_dequant_conv_dynamic(self):
+    def test_dynamic_multi_op(self):
```

Comment:
why is the name changed? this is supposed to be a unittest for insert_quant_dequant I think, you can add integration tests separately

Reply:
There was already a separate test for `test_insert_quant_dequant_linear_dynamic` so this isn't necessary. I'll change it to an e2e test.